arangod/RestHandler/RestAdminExecuteHandler.cpp

```diff
@@ -61,9 +61,16 @@ RestStatus RestAdminExecuteHandler::execute() {
 
   TRI_ASSERT(server().getFeature<V8DealerFeature>().allowAdminExecute());
 
-  std::string_view bodyStr = _request->rawPayload();
-  char const* body = bodyStr.data();
-  size_t bodySize = bodyStr.size();
+  std::string_view payload;
+  size_t bodySize;
+  if (_request->header(StaticStrings::ContentTypeHeader) == "text/plain") {
+    payload = _request->rawPayload();
+  } else {
+    VPackSlice payloadParsed = _request->payload(false);
```

Comment:
In case the content-type is not exactly `text/plain`, we will now always call the payload method, which will throw an error if the request is not JSON nor velocypack. What if there is no content-type set at all? Will this also fail now? IMO this case previously worked, but may now fail.

We are also assuming that what ever gets sent in the body parses to a JSON or velocypack string. If some other data type is sent, then the following `payloadParsed.stringView()` call will throw an exception. Is that intentionally not handled here? Does an appropriate error message come back to the caller in this case?